solr_conf/conf/managed-schema

```diff
@@ -369,10 +369,11 @@
 
     <!-- A general text field that has reasonable, generic
          cross-language defaults: it tokenizes with StandardTokenizer,
-	       removes stop words from case-insensitive "stopwords.txt"
-	       (empty by default), and down cases.  At query time only, it
-	       also applies synonyms.
+         removes stop words from case-insensitive "stopwords.txt"
+         (empty by default), down cases, and performs Unicode normalizatin.
```

Comment:
small typo here (might be mine, actually) of "normalizatin", which sounds like a fun word...we're normalizatin' all the Unicode